code/game/gamemodes/cult/talisman.dm

```diff
@@ -15,13 +15,13 @@
 
 /obj/item/paper/talisman/stun/attack_self(mob/living/user)
 	if(iscultist(user))
-		to_chat(user, "This is a stun talisman.")
+		to_chat(user, "This is a stun talisman. It can disable humans, temporarily.")
 	..()
 
 /obj/item/paper/talisman/stun/attack(mob/living/M, mob/living/user)
 	if(!iscultist(user))
 		return
-	user.say("Dream Sign: Evil Sealing Talisman!") //TODO: never change this shit
+	user.say("Dream Sign: Evil Sealing Talisman!", LANGUAGE_CULT) // this is hilarious but muh immersion!!
```

Comment:
I can't believe this is a touhou reference smh

Reply:
me too but one must carry on :trollface: